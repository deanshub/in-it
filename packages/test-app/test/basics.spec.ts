import fs from 'fs-extra'
import path from 'path'

test('stats exists', async() => {
    const statsFilePath = path.join(__dirname, '../dist/stats.json')
    expect(await fs.exists(statsFilePath)).toBe(true)
    expect(await fs.readJson(statsFilePath)).toMatchSnapshot()
})