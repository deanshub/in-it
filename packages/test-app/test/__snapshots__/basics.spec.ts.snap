// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`stats exists 1`] = `
{
  "assets": {},
  "chunks": {
    "index": {
      "modules": [
        {
          "id": "/Users/deanshub/dev/in-it/packages/test-app/build/index.js",
          "name": "index.js",
          "rendered": [],
          "size": 154,
        },
      ],
      "size": 155,
    },
  },
  "entry": "main.js",
}
`;
